data/PA/authorities.json

```diff
@@ -1,14 +1,17 @@
 {
-  "state": {
-    "pa-department-of-community-and-economic-development": {
-      "name": "Department of Community and Economic Development"
-    }
-  },
   "other": {
     "pa-first-energy": {
       "name": "First Energy"
     }
   },
+  "state": {
+    "pa-department-of-community-and-economic-development": {
+      "name": "Department of Community and Economic Development"
```

Comment:
Is there a particular reason this doesn't have the state name in it too?

Reply:
no reason - will update